bamboo/bootstrap-lint-audit.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 set -e
+
 # This script runs before lint.sh, audit.sh in the agent container
+. ./bamboo/use-working-directory.sh
 . ./bamboo/set-bamboo-env-variables.sh
 . ./bamboo/abort-if-not-pr.sh
 
+echo "***Using NPM VERSION*** $(npm --version)"
```

Comment:
Did you want to leave this? I very much don't care either way 😄

Reply:
Hmm, no, if we want to leave that we could put it in some of the boilerplate setup scripts.  I'll remove it.

Reply:
d0c4eaee01